.github/workflows/release.yml

```diff
@@ -30,14 +30,25 @@ jobs:
             git push origin $RELEASE_TAG --force
           fi
 
-      - name: Build the image
-        run: docker build --tag $IMAGE_NAME:latest .
-        
-      - name: Push the image
-        run: >
-          RELEASE_TAG="$(git describe --exact-match --tags)" && IMAGE_TAG=${RELEASE_TAG#v} &&
-          docker login --username=${{ secrets.QUAY_USERNAME }} --password=${{ secrets.QUAY_PASSWORD }} quay.io &&
-          docker tag $IMAGE_NAME:latest quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:$IMAGE_TAG &&
-          docker push quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:$IMAGE_TAG &&
-          docker tag $IMAGE_NAME:latest quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:latest &&
-          docker push quay.io/${{ secrets.QUAY_NAMESPACE }}/$IMAGE_NAME:latest
\ No newline at end of file
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to image repository quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Build and Push the image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
+          no-cache: true
+          push: true
+          tags: ${{ secrets.QUAY_NAMESPACE }}/${{env.IMAGE_NAME}}:dev.latest, $(git describe --exact-match --tags | sed 's/v//')
```

Comment:
Why is this step causing the following error on my fork?
https://github.com/brusdev/activemq-artemis-self-provisioning-plugin/actions/runs/5330753167/jobs/9657764969
```
Error: buildx failed with: ERROR: invalid tag "$(git describe --exact-match --tags | sed 's/v//')": invalid reference format
```

Reply:
> $(git describe --exact-match --tags | sed 's/v//')

I didn't see this issue on PR.
It seems  `$(git describe --exact-match --tags | sed 's/v//')` tag format is not correct. I'll check it. Can you try to build without `$(git describe --exact-match --tags | sed 's/v//')` tag.

Reply:
Sure, I'll try another build when I'll review this PR again, in the mean time you could execute this workflow on your fork to reproduce and fix the issue.

Reply:
Fixed the tag issue on fork repo.
release CI job is running on fork repo, changes will be add in PR when fork ci job will green.
https://github.com/ajaypratap003/activemq-artemis-self-provisioning-plugin/actions/runs/5336328861/jobs/9670860246